component/main.jsonnet

```diff
@@ -1,10 +1,85 @@
-// main template for prom-label-proxy
 local kap = import 'lib/kapitan.libjsonnet';
 local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
-// The hiera parameters for the component
+
 local params = inv.parameters.prom_label_proxy;
+local instance = inv.parameters._instance;
+
+local labels = {
+  'app.kubernetes.io/name': 'prom-label-proxy',
+  'app.kubernetes.io/instance': instance,
+  'app.kubernetes.io/managed-by': 'syn',
+};
 
-// Define outputs below
 {
+  deployment: kube.Deployment(instance) {
+    metadata+: {
+      namespace: params.namespace,
+      labels+: labels,
+    },
+    spec+: {
+      template+: {
+        spec+: {
+          containers_+: {
+            proxy: kube.Container('proxy') {
+              image: params.images['prom-label-proxy'].registry + '/' + params.images['prom-label-proxy'].image + ':' + params.images['prom-label-proxy'].tag,
```

Comment:
Maybe put `params.images['prom-label-proxy']` into a local variable. I assume this would make things more readable.